src/python/pants/notes/2.14.x.md

```diff
@@ -1,5 +1,95 @@
 # 2.14.x Release Series
 
+## 2.14.0.dev4 (Aug 05, 2022)
+
+### New Features
+
+* Add `[pytest].config` to set custom config file paths ([#16423](https://github.com/pantsbuild/pants/pull/16423))
+
+* java/protobuf: add support for grpc-java codegen ([#16350](https://github.com/pantsbuild/pants/pull/16350))
+
+### User API Changes
+
+* Deprecate `[python].lockfile_generator` ([#16418](https://github.com/pantsbuild/pants/pull/16418))
+
+* Replace `--[no-]process-cleanup` with `--keep-sandboxes={always,never,on_failure}` ([#16415](https://github.com/pantsbuild/pants/pull/16415))
+
+* Provide foundations for a deploy goal ([#16335](https://github.com/pantsbuild/pants/pull/16335))
```

Comment:
Hm I think this should have been Plugin API Change or Internal, as it isn't wired up to anything. Could you please move to Plugin API?